registry/directory/directory.go

```diff
@@ -42,12 +42,12 @@ import (
 	"github.com/apache/dubbo-go/remoting"
 )
 
-// Options ...
+// Options Option is configuration related  struct of serviceTTL
 type Options struct {
```

Comment:
what abount change to `Option`?

Reply:
The comment must be a complete sentence.

Reply:
No `Options Option is configuration related  struct of serviceTTL` but `Options  is configuration related  struct of serviceTTL`